RetailCoder.VBE/Settings/HotkeySetting.cs

```diff
@@ -21,10 +22,18 @@ public class HotkeySetting
         public bool HasAltModifier { get; set; }
         public bool HasCtrlModifier { get; set; }
 
+        public CommandBase Command { get; set; }
+
         [XmlIgnore]
-        public string Prompt
+        public string Prompt => RubberduckUI.ResourceManager.GetString("HotkeyDescription_" + Name, CultureInfo.CurrentUICulture);
+
+        public HotkeySetting()
+        {
+        }
```

Comment:
You should probably remove the default ctor

Reply:
It is needed for deserialization.

Reply:
I plan on adding a comment to this constructor to warn others that it shouldn't be used directly.

Reply:
Yup.

```csharp
/// <Summary>Constructor required for XML serialization.</Summary>
```